tools/report-converter/tests/unit/analyzers/tidy_output_test_files/tidy3_cpp.plist

```diff
@@ -22,38 +22,37 @@
 				<key>line</key>
 				<integer>4</integer>
 			</dict>
-			<key>path</key>
+			<key>notes</key>
 			<array>
 				<dict>
-					<key>depth</key>
+				<key>location</key>
+				<dict>
+					<key>col</key>
+					<integer>12</integer>
+					<key>file</key>
 					<integer>0</integer>
-					<key>kind</key>
-					<string>event</string>
-					<key>location</key>
-					<dict>
-						<key>col</key>
-						<integer>12</integer>
-						<key>file</key>
-						<integer>0</integer>
-						<key>line</key>
-						<integer>4</integer>
+					<key>line</key>
+					<integer>4</integer>
 					</dict>
 					<key>message</key>
 					<string>nullptr (fixit)</string>
 				</dict>
+			</array>
+			<key>path</key>
+			<array>
 				<dict>
 					<key>depth</key>
 					<integer>0</integer>
 					<key>kind</key>
 					<string>event</string>
 					<key>location</key>
 					<dict>
-						<key>col</key>
-						<integer>12</integer>
-						<key>file</key>
-						<integer>0</integer>
-						<key>line</key>
-						<integer>4</integer>
+							<key>col</key>
+							<integer>12</integer>
+							<key>file</key>
+							<integer>0</integer>
+							<key>line</key>
+							<integer>4</integer>
```

Comment:
Everywhere in the file, the indentation is 2 spaces, but here it was changed to 4 spaces, and apart from the spacing there are no changes to this part?

Reply:
No other changes. Spaces fixed.